src/app/system/manage-surveys/manage-surveys.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Surveys
+  </button>
+</div>
+  
+<div class="container">
+
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+  
+  <div class="mat-elevation-z8">
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <!-- Key Column -->
+      <ng-container matColumnDef="key">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header='key'> Key </th>
+          <td mat-cell *matCellDef="let survey"> 
+            {{ survey.key }} 
+          </td>
+      </ng-container>
+        
+      <!-- Name Column -->
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header='name'> Name </th>
+        <td mat-cell *matCellDef="let survey"> 
+          {{ survey.name }} 
+        </td>
+      </ng-container>
+  
+      <!-- Description Column-->
+      <ng-container matColumnDef="description">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header='description'> Description </th>
+        <td mat-cell *matCellDef="let survey"> 
+          {{ survey.description }} 
+        </td>
+      </ng-container>
+  
+      <!-- CountryCode Column -->
+      <ng-container matColumnDef="countryCode">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header='countryCode'> Country Code </th>
+        <td mat-cell *matCellDef="let survey"> 
+          {{ survey.countryCode }} 
+        </td>
+      </ng-container>
+  
+      <!-- Status Column -->
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header='status'> Status </th>
+        <td mat-cell *matCellDef="let survey">
+            <div [className]="!isActive(survey.validFrom, survey.validTo) === true ? 'disabled' : 'enabled'">
+                <fa-icon matTooltip="{{ !isActive(survey.validFrom, survey.validTo) === true ? 'Disabled' : 'Enabled' }}" matTooltipPosition="right" icon="circle" size="lg"></fa-icon>
+            </div>
+    
+        </td>
+      </ng-container>
+  
+      <!-- Action Column -->
+      <ng-container matColumnDef="action">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header='action'> Action </th>
+        <td mat-cell *matCellDef="let survey">
+            <a mat-raised-button [hidden]="!isActive(survey.validFrom, survey.validTo)" type="button" color="warn">Deactivate</a>
```

Comment:
Use button tag with an icon to ensure consistency with other parts of the application for example:
```html
<button mat-button color="accent">
    <fa-icon icon="lock-open"></fa-icon>&nbsp;&nbsp;Activate
</button>
<button mat-button color="warn">
    <fa-icon icon="lock"></fa-icon>&nbsp;&nbsp;Deactivate
</button>
```

Also, don't use `hidden` use `*ngIf` instead.